src/relayserver/runServer.js

```diff
@@ -22,36 +22,35 @@ const envDefaults = Object.entries(process.env)
 const argv = parseArgs(process.argv.slice(2), {
   string:
     [
-      'BaseFee',
-      'PercentFee',
-      'Url',
-      'RelayHubAddress',
-      'DefaultGasPrice',
-      'GasPricePercent',
-      'RegistrationBlockRate',
-      'EthereumNodeUrl',
-      'Workdir'
+      'config',
+      'baseRelayFee',
+      'pctRelayFee',
+      'url',
+      'relayHubAddress',
+      'defaultGasPrice',
+      'gasPricePercent',
+      'ethereumNodeUrl',
+      'workdir'
     ],
-  boolean: ['DevMode', 'Debug'],
+  boolean: ['devMode', 'debug'],
   alias: {},
   default: envDefaults
```

Comment:
current **envDefaults** will not work: it requires env var to be CamelCase (starting with capital and then small letter). 
This won't work with new names, that some are all lowercase..
simplest fix is accept any env.var starting with lowercase (all standard env.vars are all caps)
e.g.: `/^[a-z]\w*$/`

Reply:
What is envDefaults?

Reply:
fixed